packages/medusa/src/loaders/helpers/routing/__tests__/index.spec.ts

```diff
@@ -156,7 +156,7 @@ describe("RoutesLoader", function () {
     })
 
     it("should return 200 when customer is authenticated", async () => {
-      const res = await request("GET", "/store/me/protected", {
+      const res = await request("GET", "/store/customers/me/protected", {
```

Comment:
Question: are we sure we don't want to stick to store/me? wouldn't having the customer segment leak some info?

Reply:
@adrien2p no idea TBH, I think having `POST store/customers` for registration is a bit cleaner, but if you think we should change it maybe raise it in Slack?